src/components/shared/GlobalStyle/GlobalStyle.tsx

```diff
@@ -10,9 +10,21 @@ export const GlobalStyle = createGlobalStyle`
     }
   }
 
+  @font-face {
+    font-family: SDSYuGothic;
+    font-weight: 400;
+    src: local('Yu Gothic Medium');
+  }
+
+  @font-face {
+    font-family: SDSYuGothic;
+    font-weight: 700;
+    src: local('Yu Gothic Bold');
+  }
+
   body {
     margin: 0;
-    font-family: Yu Gothic Medium, 游ゴシック Medium, YuGothic, 游ゴシック体, sans-serif;
+    font-family: SDSYuGothic, Yu Gothic, YuGothic, 游ゴシック体, sans-serif;
```

Comment:
```suggestion
    font-family: SDSYuGothic, 'Yu Gothic', YuGothic, sans-serif;
```

こうでも良いかなと思いました。

## 一部をクオート`"フォント名"`で括った理由

https://triple-underscore.github.io/css-fonts-ja.html#font-family-prop

> [\<family-name>](https://triple-underscore.github.io/css-fonts-ja.html#family-name-value) によるフォントファミリ名は、 CSS 文字列（ [<string>](https://triple-underscore.github.io/css-values-ja.html#string-value) ）として，引用符で括った上で与えるか、 または［ 1 個以上の [CSS 識別子](https://triple-underscore.github.io/css-values-ja.html#css-css-identifier)が成す連列 ］として，引用符で括らずに与えなければならない。 従って，引用符で括らない場合、 各トークンの先頭にある［ 約物のほとんど／数字 ］は，エスケープされなければならない。
> Font family names other than generic families must either be given quoted as strings, or unquoted as a sequence of one or more identifiers. This means most punctuation characters and digits at the start of each token must be escaped in unquoted font family names.

> エスケープの誤記を避けるため、［ 空白／数字／［ ハイフン以外の約物 ］］を包含するようなフォントファミリ名は，引用符で括ることが推奨される：
> To avoid mistakes in escaping, it is recommended to quote font family names that contain white space, digits, or punctuation characters other than hyphens:


列挙されているフォントファミリーは、いちおうはクオートする必要はないものです。空白・数字・ハイフン以外の約物を含む文字列に関しては括ることが推奨されていたので、囲っておこうかな〜。という感じです。

## 游ゴシック体を消した理由

最近の環境においては、英語名でも和名でも認識できるようなので、あえて書く必要はないかなと思いました。

ただこれといってエビデンスはないので、残しても良いと思います。

Reply:
ありがとうございます！説明読んで、把握&賛成なのでsuggetion取り込みます！

Reply:
lintでエラーが出てしまいました。

https://github.com/kufu/stylelint-config-smarthr/blob/master/index.js#L12C4 の設定の影響だと思われます。
`always-where-required`は必要なら囲うがそれ以外では囲わないとのことなのでlintに従ってクォートは除去します。